go.mod

```diff
@@ -82,15 +82,14 @@ replace (
 
 require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/argoproj/argo-rollouts v1.1.1
 	github.com/census-instrumentation/opencensus-proto v0.3.0
 	github.com/cncf/xds/go v0.0.0-20220121163655-4a2b9fdd466b
+	github.com/datawire/ambassador-agent v0.0.7
```

Comment:
Can you run `make go-mod-tidy`? 

I would think this and its deps would be removed now that it is no longer in busyambassador or is it being referenced somewhere else?